mfr/server/static/js/mfr.js

```diff
@@ -80,6 +80,8 @@
             self.pymParent.iframe.setAttribute('allowfullscreen', '');
             self.pymParent.iframe.setAttribute('webkitallowfullscreen', '');
             self.pymParent.iframe.setAttribute('scrolling', 'yes');
+            self.pymParent.iframe.setAttribute('horizontalscrolling', 'yes');
```

Comment:
1. The `scrolling` attribute for `iframe` is not supported by `HTML5`, can this be the reason? It seems that IE11 dropped several support for `HTML4` to better support `HTML5` and `CSS`: https://msdn.microsoft.com/en-us/library/bg182625(v=vs.85).aspx.
2. Is `horizontalscrolling` and `verticalscrolling` a valid `iframe` attributes? I can't find it though.

Reply:
I don't know about the documentation or how exactly this is supported, but it does work. I tested locally in Virtual Box test environment provided by Microsoft. I would also create another ticket for the Pym update.

Reply:
Sounds good and thanks. The commit message is good enough for us to keep track of why we made this change. 👍